public/components/map_container/map_container.tsx

```diff
@@ -68,6 +85,25 @@ export const MapContainer = ({
     maplibreInstance.on('move', () => {
       return setZoom(Number(maplibreInstance.getZoom().toFixed(2)));
     });
+    const mapContainerElement: HTMLElement | null = document.querySelector('.map-page');
+    let resizeChecker: ResizeChecker;
+    if (mapContainerElement) {
+      resizeChecker = new ResizeChecker(mapContainerElement);
+      if (inDashboardMode) {
+        resizeChecker.on(
+          'resize',
+          throttle(() => {
+            maplibreInstance?.resize();
+          }, 300)
+        );
+      }
+    }
```

Comment:
Can you please create a method and add comment why you are doing this?

Reply:
By default, Maplibre only auto resize map window when browser size changes, but in dashboard mode, we need manually resize map window size when map panel size changes